src/main/java/com/dungeoncrawler/Javiarenka/character/Monster.java

```diff
@@ -32,6 +38,21 @@ public void setDamageStrength(int damageStrength) {
         this.damageStrength = damageStrength;
     }
 
+    public void saveThisMonster() {
+        Gson gson = new GsonBuilder()
+                .setPrettyPrinting()
+                .create();
+        System.out.println(this);
+        try {
+            Writer writer = new FileWriter("src/main/java/com/dungeoncrawler/Javiarenka/dataBase/monsters/" + getRace().getMonsterRaceName() + "---" + getName() + ".txt");
+            gson.toJson(this, writer);
+            writer.flush();
+            writer.close();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
```

Comment:
saveThisMonster nam by się przydał, jeśli mielibyśmy jakieś narzędzie do tworzenia własnych monsterów. Ale jeśli będziemy mieli statyczną tablicę z której będziemy do gry pobierać monstery, to ta metoda może się nam nie przydać.

Reply:
no właśnie nie byłem pewny czy potrzebujemy monstery też jako pliki jsonowe przy statycznych danych, póki co zostawiłem tę metodę ale wykomentowałem i oznaczyłem jako todo